### PR TITLE
Adding ERD & DFD

### PR DESCRIPTION
**Pull request by RonTuretzky**
#32

### CODE REVIEW

**Review by ofirtzabari — APPROVED** [review]
good job

**Review by kobihk — CHANGES_REQUESTED** [review]
- Please update the PR title to be more informative
- There is the same user for all?! which means I can join the commune and delete everyone :)  or change the expenses... IMHO you should add a manager that has extra privilege

**Review by EliasAndry — APPROVED** [review]

**Review by EliasAndry — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -67,3 +67,11 @@ Must be a list of measurable achievements that indicate the work in the issue is
...
 5. Be strict about temporary code
 6. Visualize the bigger picture
+
+## ERD
+
+![ERD](https://user-images.githubusercontent.com/74178515/204143707-970216f5-e382-47d1-ab2e-209aa7aaa18b.png)
+
+## DFD
+
+![DFD](https://user-images.githubusercontent.com/74178515/204143717-32b0ef58-5bdd-438f-9523-9184e3921f9e.png)
```
[the remark this answers is not in the document]
Comment (RonTuretzky) [reply CONTRIBUTING.md:77]: PR title amended 
There is a user table, the commune model holds a collection of users who are members, they can only suggest decisions
Adding a manager with elevated priviliges kind of defeats the purpose imo 
Moving to the read me @kobihk

**Review by FaezZangariya — APPROVED** [review]

**Review by kobihk — APPROVED** [review]

**Review by EliasAndry — APPROVED** [review]

**Review by ofirtzabari — APPROVED** [review]
LGTM

**Comment by ofirtzabari** [comment]
LGTM

**Pull request merged**